packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.encryption_extension.test.ts

```diff
@@ -529,6 +550,31 @@ describe('SavedObjectsRepository Encryption Extension', () => {
       expect(result.saved_objects.length).toBe(1);
       expect(result.saved_objects[0].error).toBeUndefined();
     });
+
+    it('allows to opt-out of random ID enforcement', async () => {
+      mockEncryptionExt.isEncryptableType.mockReturnValue(true);
+      mockEncryptionExt.shouldEnforceRandomId.mockReturnValue(false);
+      mockEncryptionExt.decryptOrStripResponseAttributes.mockResolvedValue({
+        ...encryptedSO,
+        ...decryptedStrippedAttributes,
+      });
+
+      const result = await bulkCreateSuccess(
+        client,
+        repository,
+        [{ ...encryptedSO, version: mockVersion }],
+        {
+          overwrite: true,
+          // version: mockVersion, // this doesn't work in bulk...looks like it checks the object itself?
+        }
```

Comment:
Should the override be removed here so we're not overlapping with the previous test case?

```suggestion
        {
          // version: mockVersion, // this doesn't work in bulk...looks like it checks the object itself?
        }
```

Reply:
Fixed in: https://github.com/elastic/kibana/pull/198287/commits/b84429c3700809c4254f2050eaec09ed9727bc14